misc/rapiddisk-rootdev/install_initrd.sh

```diff
@@ -38,61 +48,90 @@ is_num()  {
 
 myerror()  {
 
-	ihelp
-	echo "**** $1"
+#	ihelp
```

Comment:
Why is this commented out? Did we intend on removing it? If so, just remove it. Although, in most cases, especially if it is a result of invalid entries or a parsing error, it may just be more useful to uncomment and keep it in. Thoughts?

Reply:
Removed: the user cannot miss the error message
Uncommented: I believe the error message can be missed.

I don't know... I believe I will remove the line and let the user invoke the script with `--help` or `-h`... or maybe show the help only on specific errors.